### PR TITLE
Add parameters to vary in non-degradation comparisons

### LINKED ISSUE #16: Add parameters to vary
Add the parameters to vary in non-degradation comparisons.

### PR DESCRIPTION
**Pull request by Saransh-cpp**
Closes #16 

Added parameters to vary by taking help from the colab notebook.

### CODE REVIEW

**Comment by Saransh-cpp** [comment]
The +-10% range is not showing a lot of deviation in plots, should I increase it? (If yes then I will check what range is giving some good plots and add it in)

**Comment by Saransh-cpp** [comment]
The +-10% range is not showing a lot of deviation in plots, should I increase it? (If yes then I will check what range is giving some good plots and add it in)

**Comment by tinosulzer** [comment]
Yes, add /2 and *2

File: bot/utils/parameter_value_generator.py [hunk 1]
```diff
@@ -14,27 +23,18 @@ def parameter_value_generator(chemistry, parameter):
...
     else:
-        params = pybamm.ParameterValues(chemistry=chemistry)
-        param_value = desired_decimal_point_generator(
-            params[parameter]*0.9, params[parameter]*1.1, 2
-        )
+        base_value[parameter] = params[parameter]
+        params[parameter] = pybamm.InputParameter(parameter)
+
+    param_value = desired_decimal_point_generator(
+        (base_value[parameter] - 0.1)*0.5, (base_value[parameter] + 0.1)*2, 2
```
Comment (brosaplanella) [inline bot/utils/parameter_value_generator.py:37]: What is the reasoning behind these limits? Note that between approx 0.1 and 0.8 the first argument is larger than the second (not sure if that's a problem). Also, in the first argument you get complex values if `base_value < 0.1`.

**Review by brosaplanella — APPROVED** [review]
Looks good, just see my comment in case it is relevant.

File: bot/utils/parameter_value_generator.py (hunk 1 above)
Comment (Saransh-cpp) [reply bot/utils/parameter_value_generator.py:37]: The +-0.1 was done to ensure that if the base_value is 0 then all the generated values don't come out to be 0. I don't remember specifically but there were some parameters values that were set to 0 by default and there I was encountering this problem.

An alternative might be to just set the value of the parameter (where the default value is 0) as 0.1 (before generating the random value) and then remove the +-0.1 thing, would this be a better implementation?

Comment (rtimms) [reply bot/utils/parameter_value_generator.py:37]: Are you just changing the parameters in the list `param_to_vary_list`? I don't think any of these are zero?

Comment (Saransh-cpp) [reply bot/utils/parameter_value_generator.py:37]: Oh, weird, I did encounter some cases in the old code, might be some other issue. I'll remove the +-0.1 thing.

**Pull request merged**